dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -0,0 +1,139 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const mysql = require("mysql2");
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+const { allowedNodeEnvironmentFlags } = require("process");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+//health check
+app.get("/ping", (req, res) => {
+  res.status(200).json({ message: "pong" });
```

Comment:
해당 부분에서 return 구문을 추가하여 함수를 종료할 수 있도록 수정해주세요~!

```suggestion
  return res.status(200).json({ message: "pong" });
```